config-templates/ibm/ibm-object-storage-plugin/2.2/metadata.json

```diff
@@ -43,7 +43,7 @@
     },
     {
       "name": "supported-k8s-versions",
-      "value": "1.21,1.22,1.23,1.24,1.25"
+      "value": "1.21,1.22,1.23,1.24,1.25,1.26,1.27"
```

Comment:
@Bhagyashreek8 Please update supported OCP versions field if plugin is supported on higher OCP versions.

Also, lets remove `supported-k8s-versions` completely. I will be working on cleaning it up for other templates as well. I think we are not using it anywhere in our business logic.